src/elements/asset.js

```diff
@@ -0,0 +1,26 @@
+export default function(namespace) {
```

Comment:
I'm not exactly opposed to the idea, but also don't see how this brings any benefit. I'm hoping that this package becomes the template for new namespace packages, and I'm not sure that imposing the `use` mechanism internally for each element is a good idea. For example, the `HttpRequest` and `HttpResponse` classes expose a function that requires both a namespace and a non-registered `HttpMessagePayload` base class.